src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/6.sql

```diff
@@ -0,0 +1,1944 @@
+-- Style guide: please see: https://github.com/ktaranov/sqlserver-kit/blob/master/SQL%20Server%20Name%20Convention%20and%20T-SQL%20Programming%20Style.md
+
+/*************************************************************
+    Schema Version
+**************************************************************/
+
+INSERT INTO dbo.SchemaVersion
+VALUES
+    (6, 'started')
+
+GO
+
+/*************************************************************
+    Model tables
+**************************************************************/
+
+CREATE TABLE dbo.SearchParam
+(
+    SearchParamId smallint IDENTITY(1,1) NOT NULL,
+    Uri varchar(128) COLLATE Latin1_General_100_CS_AS NOT NULL,
+    Status varchar(10) NULL,
```

Comment:
>Status [](start = 4, length = 6)

Is it mapped to `SearchParameterStatus` ?
Wouldn't be better to map it to int instead of string?

Reply:
Good question, I agree that an `int` would be smaller and more efficient space-wise. I based this implementation off of the stored procedures that I wrote for SQL export operations, where status information is stored as a `string`.

I originally had export status info serializing to an `int`, but the point was made that we serialize status values as a `string` in Cosmos DB and it would be better to do that for SQL as well. [Here ](https://github.com/microsoft/fhir-server/pull/812#discussion_r368082167)is a link to the review comment about this -- let me know what you think.

Reply:
I'm not quite understand reasoning, to be honest, mostly because I'm not sure what is 'BE' mean in that comment.
Overall, personally in spherical horse in a vacuum situation, I would prefer to have ints, but 
a) it's not that important, there is not much savings from db side considering search params would be max 10k records or so
b) you have precedent on your side, and I like to keep things align, so it's better to keep code align than do micro optimizations.

Reply:
Thanks for the reply - I agree with both your points a) and b).

Double checked with @jackliums, and by "BE" he just meant the FHIR service.

One thing I might add is if you look at the commit where I made the change to store enum values as `strings` ([link](https://github.com/microsoft/fhir-server/pull/812/commits/255ef52dc506b5e94a8fd95d21f8792de331940d)), someone reading the SQL schema file won't need to know that the `int` value `2` maps to the second `OperationStatus` enum value, `Running`.